doc/index.rst

```diff
@@ -277,11 +277,12 @@
             <div class="span4">
                 <h4>Community</h4>
                 <ul>
-                <li><em>About us</em> See <a href="about.html">authors</a> # scikit-learn</li>
+                <li><em>About us</em> See <a href="about.html#people">authors</a></li>
                 <li><em>More Machine Learning</em> Find <a href="related_projects.html">related projects</a></li>
-                <li><em>Questions?</em> See <a href="http://stackoverflow.com/questions/tagged/scikit-learn">stackoverflow</a> # scikit-learn</li>
+                <li><em>Questions?</em> See <a href="faq/">FAQ</a> and <a href="http://stackoverflow.com/questions/tagged/scikit-learn">stackoverflow</a> #scikit-learn</li>
                 <li><em>Mailing list:</em> <a href="https://lists.sourceforge.net/lists/listinfo/scikit-learn-general">scikit-learn-general@lists.sourceforge.net</a></li>
                 <li><em>IRC:</em> #scikit-learn @ <a href="http://webchat.freenode.net/">freenode</a></li>
+                <li><em>Want to contribute?</em> See <a href="developers/">developers' guide</li>
```

Comment:
I'd like to try to avoid adding somethign to the front page. We know from previously that each time we add something to the front page, it clogs it, and ideally we should be removing something. I think that your addition to the dropdown menu is "enough".